.make/test.mk

```diff
@@ -274,6 +274,12 @@ coverage-all-html: prebuild-check clean-coverage-overall $(COV_PATH_OVERALL)
 	@go tool cover -html=$(COV_PATH_OVERALL)
 
 # Experimental:
+#
+.PHONY: coveralls
+## Uploads the coverage to coveralls.io
+coveralls: $(GOVERALLS_BIN)
+	cp $(TMP_PATH)/coverage.mode-* profile.cov
+	$(GOVERALLS_BIN) -coverprofile=profile.cov -repotoken=TcANY144GHGlWbepmrjNV4TAn9Hmsm23t -service=jenkins
```

Comment:
@kwk Ah. Ok.

I have used that service sometimes back for one of my personal Go project:
https://coveralls.io/github/baijum/usfm?branch=master
It was helpful for me to see the coverage after every push.